portfolio/src/main/java/com/google/sps/data/EntityCreation.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+// TODO: add javadoc
```

Comment:
Nit: Since javadoc is just text, not code, maybe we can write it now instead of TODO.

Reply:
Fixed